kinto/core/storage/postgresql/__init__.py

```diff
@@ -74,34 +74,41 @@ def __init__(self, client, max_fetch_size, *args, **kwargs):
         self._max_fetch_size = max_fetch_size
 
     def _execute_sql_file(self, filepath):
-        here = os.path.abspath(os.path.dirname(__file__))
-        schema = open(os.path.join(here, filepath)).read()
+        schema = open(filepath).read()
         # Since called outside request, force commit.
         with self.client.connect(force_commit=True) as conn:
             conn.execute(schema)
 
-    def initialize_schema(self):
+    def initialize_schema(self, is_dry=False):
         """Create PostgreSQL tables, and run necessary schema migrations.
 
         .. note::
 
             Relies on JSONB fields, available in recent versions of PostgreSQL.
         """
+        here = os.path.abspath(os.path.dirname(__file__))
+
         version = self._get_installed_version()
         if not version:
+            filepath = os.path.join(here, 'schema.sql')
+            if is_dry:
+                logger.info("Create PostgreSQL storage schema at version "
+                            "%s from %s" % (self.schema_version, filepath))
```

Comment:
I believe that this message should not be within `is_dry` but always in output instead